packages/frontend/src/pages/Periods/Details/PeriodDetailsPage.tsx

```diff
@@ -109,6 +110,13 @@ export const PeriodDetailsPage = (): JSX.Element | null => {
                 replace
                 rounded
               />
+              <NavItem
+                to={`${url}/givers`}
+                description="Givers"
+                icon={faHandsHoldingCircle}
```

Comment:
I suggest we use 'hand-holding-heart' for giver and 'hands-holding-heart' for receiver.

Reply:
@kristoferlund hm, looks like `hands-holding-heart` is a pro icon